File: dotnet/System/Workspace/Adapters/Allors.Workspace.Adapters.Remote/Database/Push/PushResult.cs
// <copyright file="RemotePushResult.cs" company="Allors bv">
// Copyright (c) Allors bv. All rights reserved.
// Licensed under the LGPL license. See LICENSE file in the project root for full license information.
// </copyright>

namespace Allors.Workspace.Adapters.Remote
{
    using Allors.Protocol.Json.Api;

    public class PushResult : Result, IPushResult
    {
        public PushResult(ISession session, Response response) : base(session, response)
        {
        }
    }
}

File: typescript/libs/workspace/domain/json/rxjs/system/src/lib/database/push/PushResult.ts
import { PushResponse } from '@allors/protocol/json/system';
import { IPushResult, ISession } from '@allors/workspace/system';
import { Result } from '../Result';

export class PushResult extends Result implements IPushResult {
  constructor(session: ISession, response: PushResponse) {
    super(session, response);
  }
}
